.github/workflows/update-release-draft.yml

```
@@ -1,18 +1,71 @@
 # Automates creation of Release Drafts using Release Drafter
-name: Update Release Draft
+name: Update Release Draft & Create Release Bump PR
```

```suggestion
name: Update Release Draft
```

I think this repo might have a different name from all the others?

---

I would not be too sure of that :sweat_smile: